pkg/config/config.go

```
@@ -0,0 +1,29 @@
+package config
+
+import (
+	"github.com/spf13/viper"
+	"log"
+	"path"
+	"path/filepath"
+	"strings"
+)
+
+//Load is a function that set configuration
+func Load(configStruct interface{}, configPath string) error {
+	dir, file := filepath.Split(configPath)
+	ext := path.Ext(file)
+	fileExt := strings.TrimPrefix(ext, ".")
+	fileName := strings.TrimSuffix(file, ext)
+
+	viper.SetConfigName(fileName)
+	viper.SetConfigType(fileExt)
+	viper.AddConfigPath(dir)
+	err := viper.ReadInConfig()
+	if err != nil {
+		log.Fatalf("Error config file: %s \n", err)
```

please return an error instead of logging it.